pom.xml

```diff
@@ -1985,6 +1986,13 @@
                 <version>0.9.35</version>
             </dependency>
 
+            <!-- TODO: update in airbase -->
+            <dependency>
```

Comment:
Set the `dep.slf4j.version` property instead, so that all of the SLF4J dependencies have the same version